src/kvstore/test/NebulaStoreTest.cpp

```diff
@@ -760,13 +760,13 @@ TEST(NebulaStoreTest, ThreeCopiesCheckpointTest) {
         fs::FileUtils::remove(folly::stringPrintf("%s/data", rm.data()).c_str(), true);
         fs::FileUtils::remove(folly::stringPrintf("%s/wal", rm.data()).c_str(), true);
         std::string mv = folly::stringPrintf(
-                "/usr/bin/mv %s/disk%d/nebula/0/checkpoints/snapshot/data %s/disk%d/nebula/0/data",
+                "mv %s/disk%d/nebula/0/checkpoints/snapshot/data %s/disk%d/nebula/0/data",
                 rootPath.path(), i , rootPath.path(), i);
         sleep(1);
         auto ret = system(mv.c_str());
```

Comment:
Consider the function `rename` in stdio.h?

Reply:
`std::rename` is for files only, not directories.
Actually, I'm more optimistic with `std::filesystem::rename` . However, the current environment does not yet allow for the use of `std::filesystem::rename`, which will take some time.

Reply:
The `rename` could apply to folder according to the man page. And I tried in nebula-dev containers.

Reply:
> The `rename` could apply to folder according to the man page. And I tried in nebula-dev containers.

Aha，Thanks for your verification ! Would you like to provide the PR？

Reply:
> > The `rename` could apply to folder according to the man page. And I tried in nebula-dev containers.
> 
> Aha，Thanks for your verification ! Would you like to provide the PR？

Ok.

Reply:
Close this one. And PR #1387 will  fix this bug.